plugins/quay/playwright.config.ts

```diff
@@ -0,0 +1,33 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```

Comment:
will this config be at each plugin level or can it be at the app level as this would be similar for most plugins?

Reply:
It is going to be similar for most plugins. I've put it at the plugin level since it was indicated to me that plugins might change code bases in the future to make it self contained and easy to move.